fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -1590,6 +1594,70 @@ public ParseNode visitDeleteStatement(StarRocksParser.DeleteStatementContext con
     }
     // ------------------------------------------- Routine Statement ---------------------------------------------------
 
+    @Override
+    public ParseNode visitCreateRoutineLoadStatement(StarRocksParser.CreateRoutineLoadStatementContext context) {
+        String database = context.db.getText();
+        String name = context.name.getText();
+        String tableName = context.table.getText();
+
+        List<ParseNode> loadPropertyList = new ArrayList<>();
+        List<StarRocksParser.LoadPropertiesContext> loadPropertiesContexts = context.loadProperties();
```

Comment:
loadPropertiesContexts may be null because it is optional. should check it and add an UT to test it.

Reply:
[check loadPropertiesContext is not null](https://github.com/StarRocks/starrocks/pull/9981/commits/1d4f8f023a78a81869a20f6c9c5b578bba9d3abd)